js/util.js

```diff
@@ -41,15 +41,15 @@
     element.classList.add(Style.HIDE);
   };
 
-  var deleteElement = function (element) {
+  var removeElement = function (element) {
     element.remove();
   };
 
   var setDisabled = function (element) {
     element.disabled = true;
   };
 
-  var deleteDisabled = function (element) {
+  var resetDisabled = function (element) {
```

Comment:
Почему `reset`?

Эта функция не знает, какое значение `disabled` было установлено прежде, а `false` не является значением по умолчанию для всех элементов. Тем более уже имеется функция с обратным действием `unsetDisabled`:

https://github.com/htmlacademy-javascript/834173-keksobooking-18/blob/5f631625593046b5d62f71e10f58ee2fa7eb361c/js/util.js#L48

```suggestion
  var unsetDisabled = function (element) {
```